apps/src/maze/maze.js

```diff
@@ -1170,9 +1168,14 @@ Maze.scheduleAnimations = function (singleStep) {
     }
 
     animateAction(actions[index], singleStep, timePerAction);
+
+    var command = actions[index] && actions[index].command;
+    var timeModifier = (skin.actionSpeedScale && skin.actionSpeedScale[command]) || 1;
+    var timeForThisAction = parseInt(timePerAction / timeModifier, 10);
```

Comment:
I think that elsewhere our modifiers are multipliers rather than divisors. To be more consistent, we might make this value 0.5 in the skin and multiply instead.

Also, we shouldn't need to parseInt here as timePerAction and timeModifier should already both be ints.